Infrastructure.DataAccess/Infrastructure.DataAccess.csproj

```diff
@@ -624,6 +624,10 @@
     <Compile Include="Migrations\202009281002528_DPR_Transfer_To_Third_Countries.designer.cs">
       <DependentUpon>202009281002528_DPR_Transfer_To_Third_Countries.cs</DependentUpon>
     </Compile>
+    <Compile Include="Migrations\202009300908424_Added_Agreement_Oversight_Interval_Option_And_Note.cs" />
```

Comment:
Den hedder også note her. Det skulle gerne være Remark i stedet. Sørg for at lave den navne ændring når du merger master ind og alligevel skal have lavet ny migration